python/huggingfaceserver/huggingfaceserver/vllm/utils.py

```diff
@@ -54,6 +54,14 @@ def infer_vllm_supported_from_model_architecture(
 def maybe_add_vllm_cli_parser(parser: ArgumentParser) -> ArgumentParser:
     if not _vllm:
         return parser
+    parser.add_argument(
+        "--max-log-len",
```

Comment:
I don't quite understand the logic behind what config parameters `vllm` defines as a part of `EngineArgs/AsyncEngineArgs.add_cli_args(...)` versus what arguments are added externally (e.g. `max-log-len`). 

But irregardless, I can't think of a better way of adding support of this argument for now

Reply:
It seems to be defined on the AsyncEngine
https://docs.vllm.ai/en/v0.4.2/models/engine_args.html#async-engine-arguments

Reply:
Not anymore since `v0.5.3`
https://docs.vllm.ai/en/v0.5.3/models/engine_args.html